apps/authentication-portal/src/main/webapp/backupCode.jsp

```diff
@@ -0,0 +1,153 @@
+<%-- ~ Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved. ~ ~ WSO2 Inc. licenses this file to you
```

Comment:
Add the proper license.

Reply:
https://github.com/wso2/identity-apps/blob/master/apps/authentication-portal/src/main/webapp/domain.jsp#L1